types/expo/index.d.ts

```diff
@@ -2561,13 +2699,14 @@ export class Svg extends Component<{ width: number, height: number, viewBox?: st
  * Take Snapshot
  */
 export function takeSnapshotAsync(
-    view?: (number | React.ReactElement<any>),
+    node: number | React.ReactElement<any> | React.RefObject<any>,
     options?: {
-        width?: number,
-        height?: number,
-        format?: 'png' | 'jpg' | 'jpeg' | 'webm',
-        quality?: number,
-        result?: 'file' | 'base64' | 'data-uri',
+        width?: number;
+        height?: number;
+        format: 'png' | 'jpg' | 'raw' | 'webm';
```

Comment:
How about adding JSDoc comments for `format` noting that `png` is default and `webm` is only supported on Android?

Reply:
I would have added them, but most of the types here don't have any comments and it's a lot of work to add comments for everything

Reply:
Personally I think it would have been okay to just add a few documentation comments, but I certainly appreciate the willingly to include them everywhere. 🙂